code/src/layouts/cart-drawer/CartDrawer.scss

```diff
@@ -0,0 +1,50 @@
+@import "@design-system";
+
+.cart-drawer {
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  height: 100svh;
+
+  &__header {
+    padding: spacing(5) spacing(2);
+    position: relative;
+    background-color: var(--spa-black-background-color, $black);
+    box-sizing: border-box;
+  }
+
+  &__close-button {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    color: var(--spa-black-background-color, $white);
+  }
+
+  &__title {
+    font-size: $fs-lg;
```

Comment:
I suggest setting `subtitle2` (it uses 20px size)  variant to the `AppTypography` and if you need bold for this text, you can use `fontWeight` prop which includes `extra-bold` variant. For semantic you can use `h2` component

Reply:
Thanks, I used suggested approach in this case